### PR TITLE
2139 fix home page design and skeleton structure

### PR DESCRIPTION
**Pull request by Anishali2**
What I've done 

- Home Page Nav Bar Responsive and Alignmen
- Table Header Design Fix
- Timer Design

### CODE REVIEW

File: apps/web/lib/features/team/user-team-card/task-skeleton.tsx [hunk 1]
```diff
@@ -43,29 +43,23 @@ export function InviteUserTeamSkeleton() {
...
-					{t('common.TODAY')}
-				</div>
-			</Tooltip>
+		<div className="hidden sm:flex row font-normal justify-between  dark:text-[#7B8089] p-4 ml-2">
+			<p className="w-1/12">{t('common.STATUS')}</p>
+			<p className="w-2/12">{t('common.NAME')}</p>
+			<p className="w-3/12">{t('common.TASK')} </p>
+			<div className="w-2/12 text-right  ">
+				<Tooltip label={t('task.taskTableHead.TOTAL_WORKED_TODAY_HEADER_TOOLTIP')}>
+					{t('task.taskTableHead.TASK_WORK.TITLE')} {t('common.TASK')}
```
Comment (evereq) [inline apps/web/lib/features/team/user-team-card/task-skeleton.tsx:52]: @Anishali2 how you replaced that <br>?

File: apps/web/lib/features/team/user-team-card/task-skeleton.tsx [hunk 2]
```diff
@@ -43,29 +43,23 @@ export function InviteUserTeamSkeleton() {
...
-					{t('common.TASK')}
-				</div>
-			</Tooltip>
-			<div className="w-1"></div>
-			<div className=" text-center w-1/5 lg:px-3 2xl:w-52 3xl:w-64">{t('common.ESTIMATE')}</div>
-			<div className="w-1"></div>
-			<Tooltip label={t('task.taskTableHead.WORKED_ON_TASK_HEADER_TOOLTIP')}>
-				<div className="w-1/5 lg:px-3 2xl:w-52 3xl:w-64 text-center">
-					{t('task.taskTableHead.TOTAL_WORK.TITLE')}
-					<br />
```
Comment (evereq) [inline apps/web/lib/features/team/user-team-card/task-skeleton.tsx:65]: @Anishali2 how you replaced that <br>?

File: apps/web/lib/features/timer/timer.tsx [hunk 3]
```diff
@@ -58,43 +58,51 @@ export function Timer({ className }: IClassName) {
...
-							<TimerRunningSourceIcon source={timerStatus?.lastLog?.source || TimerSource.TEAMS} />
-						</Tooltip>
-					)}
+							{pad(hours)}:{pad(minutes)}:{pad(seconds)}
+							<span className="text-sm">:{pad(ms_p)}</span>
+						</Text.Heading>
+
+						<ProgressBar width="100%" className="mt-2" progress={`${activeTaskEstimation}%`} />
+					</div>
+					<div className="">
```
Comment (evereq) [inline apps/web/lib/features/timer/timer.tsx:86]: className=""?? @Anishali2

File: apps/web/lib/features/team/user-team-card/task-skeleton.tsx (hunk 1 above)
Comment (Anishali2) [reply apps/web/lib/features/team/user-team-card/task-skeleton.tsx:52]: Sir using width in pixel is a good thing but in this senario i personally prefer using col-12 architecture from bootstrap so i just did what needed to be done also this is way more responsive and readable and we don't have to condition what should be on xl , lg , md , sm etc stuff like that

File: apps/web/lib/features/team/user-team-card/task-skeleton.tsx (hunk 2 above)
Comment (Anishali2) [reply apps/web/lib/features/team/user-team-card/task-skeleton.tsx:65]: Sir I cannot remove something in this page I just created the responsive ness from scratch and put the under code in there because it is more easy , fast , and relaible 
also understanding every single class and change will take a lot of time.

File: apps/web/lib/features/timer/timer.tsx (hunk 3 above)
Comment (Anishali2) [reply apps/web/lib/features/timer/timer.tsx:86]: considered!

**Comment by Anishali2** [comment]
image

File: apps/web/lib/features/auth-user-task-input.tsx [hunk 4]
```diff
@@ -23,40 +23,39 @@ export function AuthUserTaskInput({ className }: IClassName) {
...
-			<div className="flex flex-col justify-between gap-5 lg:flex-row lg:items-center">
-				<div className="flex mb-4 lg:mb-0">
-					<span className="font-normal text-gray-500">{t('common.ESTIMATE')}:</span>
+			<div className="flex gap-5 flex-row lg:items-center justify-between ml-2">
+				<div className="xl:flex mb-4  lg:mb-0">
+					<span className="font-normal  text-gray-500 pr-2">{t('common.ESTIMATE')}:</span>
 					<TaskEstimate />
 				</div>
 
-				<div className="justify-end flex-1 hidden gap-2 md:flex">
```
Comment (evereq) [inline apps/web/lib/features/auth-user-task-input.tsx:33]: @Anishali2 can it be that "hidden" was here on purpose to hide those elements initially and show only when needed? I am not sure, just asking?

File: apps/web/lib/features/task/task-estimate.tsx [hunk 5]
```diff
@@ -50,7 +49,7 @@ export function TaskEstimate({ _task, onCloseEdition, className, loadingRef, clo
 	return (
 		<div className={clsxm('flex items-center space-x-1', className)} ref={targetEl}>
 			<TimeInputField
-				value={value['hours']}
+				value={value['hours'] ?? '00'}
```
Comment (evereq) [inline apps/web/lib/features/task/task-estimate.tsx:52]: @Anishali2 I am not sure we want default value to be 00? I.e. for user might be more fast to start type say 23 instead of need to remove those "00" symbols first, what do you think? Please make sure such "logic" changes are done in separate PRs :)

File: apps/web/lib/features/team/user-team-card/task-skeleton.tsx (hunk 1 above)
Comment (evereq) [reply apps/web/lib/features/team/user-team-card/task-skeleton.tsx:52]: Sorry, maybe I was not clear enough, it was "<br />" there in code (see line 55 in old code file) and you removed it, is it make different look? Because idea was that TITLE and TASK go on different lines, how it works now, after you removed `<br>`?

File: apps/web/lib/features/task/task-estimate.tsx (hunk 5 above)
Comment (Anishali2) [reply apps/web/lib/features/task/task-estimate.tsx:52]: Sir it's in the design that if there is no value defined then it shows 00

Comment (Anishali2) [reply apps/web/lib/features/task/task-estimate.tsx:52]: I can remove that

File: apps/web/lib/features/auth-user-task-input.tsx (hunk 4 above)
Comment (Anishali2) [reply apps/web/lib/features/auth-user-task-input.tsx:33]: Not sure because I don't change that hide and show css it's default but I'll check and fix that

Comment (evereq) [reply apps/web/lib/features/auth-user-task-input.tsx:33]: @Anishali2 I think you forgot to check / fix that, I put back that "hidden" class, hope it makes sense?

**Comment by evereq** [comment]
@Anishali2 please fix also DeepScan issues in this PR, thanks!
